### ISSUE TITLE
Autopilot not working

### ISSUE THREAD

**Issue by lobsterpub** [biscottigelato/SwiftLightning#14]
I engaged autopilot and force stopped the app, but autopilot still not working.  I have to open channels manually.

**Comment by biscottigelato** [comment]
Hi @lobsterpub. Can you detail how you enabled autopilot?

**Comment by lobsterpub** [comment]
Within the app I enabled it, then forced stopped the app, restarted and waited, but no channels were created after several days.  Also, the sync process would always stay at 99%.

**Comment by biscottigelato** [comment]
Hi, I see 2 symptoms described

1. Sync progress at 99% - Primary cause is because both the neutrino nodes that I run is down, and a 3rd one operated by Lightning Labs doesn't seem to be helping the wallet to sync with blocks. Both neutrino nodes are now backup. Sync might still take several minutes to catch up from since where it was last synced. But it should eventually complete.

2. Autopilot - Make sure you choose 'Engage Autopilot' and then click 'Apply'. At which point there's no way to proceed without hard restarting the app. Once restarted, if the wallet is fully synced with sufficient lightning nodes discovered, LND (Lightning Node powering the wallet) should start to establish channels against nodes it deemed good for creating channels with.

**Comment by tamez** [comment]
Hi @biscottigelato - I downloaded the Test Pilot app and am syncing, but have been stuck on 99% for about 1 hour. Is this a neutrino node issue again? 

If there is an issue with neutrino nodes, do you or LN need us to set up node(s)?

**Comment by biscottigelato** [comment]
Yeah, I was trying to move the btcd node to a newer version because I was having problem working with the tip of lnd. Now everything is broken. I'm trying to find a combo that will bring things back online again.

**Comment by tamez** [comment]
Ok looking forward to running the app once you are back online

**Comment by JediWed** [comment]
Any news here? Autopilot still does not work for me :(

**Comment by mattrolightning** [comment]
Autopilot still not working for me either.

**Comment by biscottigelato** [comment]
After migrating to LND v0.5, I have been seeing it keep re-doing getting compact filters. Even after it reached the highest block, it just goes back to some random block height and restart getting compact filters... It seems to cause trouble with discovering nodes and channels, making Autopilot difficult because LND needs to have an understanding of the network before it can determine which LN node to open channels against.

I think the enhancement here might be to keep LND disabled until a certain number of nodes have been discovered.
